src/config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   secret: 'thisisthebiggestsecretyouhaveeverknowninyourwholelife',
   scoreIncrementor: 10,
-  scoreDecrementor: 5
+  scoreDecrementor: 5,
+  saltRounds: 10
```

Comment:
maybe add a trailing comma to prevent unpretty diffs in the future. it is a convention in games. so just struck me to harrass you also =)